src/config.js

```diff
@@ -1,12 +1,14 @@
 export default {
   app: {
-    logoUrl: process.env.APP_LOGO,
+    title: process.env.APP_TITLE || 'Chill dashboard',
+    logoUrl: process.env.APP_LOGO || require('../public/images/chill.png'),
     logoHeight: process.env.APP_LOGO_HEIGHT || '80px'
   },
   api: {
     baseUrl: process.env.API_ENDPOINT || 'http://localhost:8000',
     endpoints: {
       status: '/api/status'
-    }
+    },
+    websocketEndpoint: process.env.WEBSOCKET_ENDPOINT || 'ws://localhost:8080'
```

Comment:
Can we make it something like this? 

```
  api: {...},
  websocket: {
    endpoint: process.env.WEBSOCKET_ENDPOINT || 'ws://localhost:8080'
  }
```
In this way we can have other params inside `websocket` too.